### ISSUE TITLE
Add edge detection to OverflowMenu to make it accessible under 200% zoom 

### ISSUE THREAD

**Issue by yumeforever** [carbon-design-system/carbon#10178]
Following up on our [conversation on Slack](https://ibm-cloudplatform.slack.com/archives/C0M053VPT/p1637055456241700), I'm creating this issue to add an Edge Detection to the Overflow menu so when users with disabilities use a 200% zoom, the content can still be scrollable and readable. This issue came up with the Data Table setting icon.
- [Issue in Cloud PAL](https://github.ibm.com/ibmcloud/pal/issues/4829)

<img width="721" alt="CleanShot 2021-11-26 at 20 23 10@2x" src="https://user-images.githubusercontent.com/3612533/143623319-8bbe50fa-114d-4e46-a2c9-9f889f21cb8d.png">
<img width="718" alt="CleanShot 2021-11-26 at 20 23 00@2x" src="https://user-images.githubusercontent.com/3612533/143623323-45d88914-bc3c-4faa-b1a0-c4449e147331.png">

**Comment by abbeyhrt** [comment]
@joshblack Is the new `Disclousre` component going to be used in `OverflowMenu`? If so, then that might cover this enhancement.

**Comment by joshblack** [comment]
@abbeyhrt sorry for the delay, the goal is for `Popover` to handle edge detection with its "auto" alignment option. `OverflowMenu` can use that component to support this use-case 👍

**Comment by mashenka123** [comment]
@joshblack Could you please help confirm when this could be fixed? Thank you!

**Comment by mashenka123** [comment]
@joshblack According to your reply in Slack---This will get fixed as a part of our v11 release, which we will be shipping at the end of March.

**Comment by mashenka123** [comment]
@joshblack Could you please confirm if this is fixed?

**Comment by tay1orjones** [comment]
It is outstanding, https://github.com/carbon-design-system/carbon/issues/10716

**Comment by mashenka123** [comment]
@joshblack Could you please help confirm when this could be fixed?

**Comment by joshblack** [comment]
Hi @mashenka123! 👋 For planning, @tay1orjones can help you out with sharing when something is scheduled. I don't believe this issue is scheduled for a sprint but will defer to Taylor 👍

**Comment by tay1orjones** [comment]
@mashenka123 This issue is dependent on #10716 and won't be addressed or scheduled until that is available.

**Comment by abbeyhrt** [comment]
We did some exploration around this and found that out of the box, OverflowMenu works as intended when inside a Modal (example [here](https://stackblitz.com/edit/github-nt5ukj?file=src/App.jsx)) It seems like the customization that has been added to the OverflowMenu in EnhancedDataTable is causing that problem at 200% zoom and I don't think that edge detection is the solution. No matter the orientation, the OverflowMenu items would still be out of view. 

If there are any plans to update to V11, we would recommend using the new Popover component for the "settings" menu in EnhancedDataTable. It supports auto alignment and will have a [tab tip variant](https://github.com/carbon-design-system/carbon/issues/11038) coming soon that should make it better suited for that use case. 

Closing this as we will not be pursuing edge detection on OverflowMenu as a solution to this problem.
